### DISCUSSION TITLE
keplergl vs deckgl

### DISCUSSION THREAD

**Discussion by manoj-1999** [visgl/deck.gl#7529] [General]
Hi,I would like to know if kepler gl gives any performance boost compared to deck gl when dealing with large data or is it just for extra customizable layers?

**Comment by ibgreen** [comment]
In principle, no. kepler.gl is built on deck.gl and uses deck.gl layers under the hood. It adds data loading and management, and a UI to allow non-programmers to work with deck.gl.
